packages/web/src/pages/rewards-page/components/ClaimAllPanel.tsx

```diff
@@ -0,0 +1,182 @@
+import { useCallback } from 'react'
```

Comment:
what if we called it `ClaimAllRewardsPanel`? I'm going to switch to this name on mobile too.